### DISCUSSION TITLE
No more blinking library icon during local database update

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#625] [Bug Reports]
If I perform a local "Update database" in the library view I see a "Done" at the end only. In earlier releases the library icon in the upper left corner was blinking during the complete database update additionally. The absence ot this function has the consequence to stare on the screen during a time consuming update of a bigger subtree not to miss the two seconds "Done" message.

**Comment by cmh714** [comment]
the little blinky in the Volume knob is gone too...

**Comment by EarsofTin** [comment]
My library icons blink until the database update is complete.  Both the upper left and the lower bar icons.  I'm on 20211114 update using multiple browsers including Firefox and Brave.  I disabled and don't see volume controls.

**Comment by cmh714** [comment]
ok spoke too soon, the blinky is in the volume knob.....yeah I dont use the volume either but its there :)
